### ISSUE TITLE
 Error "message.recipients.to must be of type Array."

### ISSUE THREAD

**Issue by Komsomol** [Freyb/strapi-provider-email-azure#1]
New installation of Strapi. When sending an Email using this I get the following error:

I know the email service works as I tested it in a separate node project using Azure only. 

```
error: Couldn't send test email: Error "message.recipients.to must be of type Array." occurred in serializing the payload - undefined..
Error: Couldn't send test email: Error "message.recipients.to must be of type Array." occurred in serializing the payload - undefined..
```

The package.json

```
    "@strapi/plugin-i18n": "4.10.5",
    "@strapi/plugin-users-permissions": "4.10.5",
    "@strapi/strapi": "4.10.5",
    "better-sqlite3": "8.0.1",
    "strapi-provider-email-azure": "^1.0.1"
```

<img width="1263" alt="Screenshot 2023-05-23 at 15 48 35" src="https://github.com/Freyb/strapi-provider-email-azure/assets/452300/914b4c92-c0b9-485d-b2c4-d96e518b6161">

**Comment by Komsomol** [comment]
Fixed this by changing how the data is formatted in recipents

`   recipients: {
            to: [
              {
                address: options.to
              }
            ],
            cc: options.cc,
            bcc: options.bcc,
          },`

**Comment by Komsomol** [comment]
Created a pull request. Tested this on Strapi and it worked.

**Comment by Komsomol** [comment]
Hey @Freyb can my pull request be merged please?

**Comment by Freyb** [comment]
Hey @Komsomol ,
Thanks for reaching out.

I don't think the solution you've provided is the correct one. As I wrote in the docs of the provider, `options.to`, `options.cc` and `options.bcc` can take up more complex forms than a simple email address.

Nevertheless, thank you for pointing out that the Strapi test email service only gives a single string to the assigned provider.
I've updated the provider to handle this case while keeping the former functionality.
